tests/fixtures/disallow-multiple-var-decl/good/for.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable ember-suave/prefer-destructuring */
```

Comment:
Hmm, I don't believe the old custom rule for object destructuring complained about this, though I guess it makes sense that it should per the rule.